pkg/plugin/conftest/plugin.go

```diff
@@ -187,6 +188,9 @@ func (p *plugin) getPolicies(ctx starboard.PluginContext) ([]string, error) {
 		if !strings.HasPrefix(key, "conftest.policy.") {
 			continue
 		}
+		if !strings.HasSuffix(key, ".rego") {
```

Comment:
Conftest will skip a policy file without the .rego suffix so there's no point in mounting it